### ISSUE TITLE
need FlyTo to ask for the height frame

### ISSUE THREAD

**Issue by tridge** [ArduPilot/MissionPlanner#3294]
The FlyTo dialog box on the map currently assumes GLOBAL_RELATIVE_ALT:
https://github.com/diydrones/MissionPlanner/blob/master/ExtLibs/ArduPilot/Mavlink/MAVLinkInterface.cs#L4344
we need the dialog to ask the user what frame they want to use "Terrain Relative", "Home Relative" or "Absolute" and pass that to the setPositionTargetGlobalInt() message.
I just don't know how to create a dialog like that with a float input and a selection drop down with the 3 choices, I will need some help from a C# developer
Anyone flying BVLOS really needs this change

**Comment by tridge** [comment]
the use of the old setWP() interface for ArduPlane should also be removed, all fw supports setPositionTargetGlobalInt()

**Comment by EosBandi** [comment]
I can work on it only after mid-March if nobody takes it till that.

**Comment by robertlong13** [comment]
I got it. Gonna try to knock it out this weekend.

**Comment by EosBandi** [comment]
Ohh, I forgot about this one. Thanks Bob. 
One question, is it possible to replace the old setWP method with  setPositionTargetGlobalInt in this or it is a scope of another PR?

**Comment by robertlong13** [comment]
Clearly it got away from me too. [checks calendar] apparently Oct 16th is not "the weekend following Feb 23"

> One question, is it possible to replace the old setWP method with setPositionTargetGlobalInt in this or it is a scope of another PR?

Ah, I missed that bit. Will need to be another PR because Michael has already merged that one. If you're up for it that would be great.
